TileStache/Goodies/VecTiles/transform.py

```diff
@@ -291,46 +291,50 @@ def road_classifier(shape, properties, fid, zoom):
 def road_sort_key(shape, properties, fid, zoom):
     # Note! parse_layer_as_float must be run before this filter.
 
-    # Calculated sort value is in the range 0 to 39
-    sort_val = 0
+    floor = 300
+    ceiling = 447
+    sort_val = floor
 
-    # Base layer range is 15 to 24
     highway = properties.get('highway', '')
     railway = properties.get('railway', '')
     aeroway = properties.get('aeroway', '')
     aerialway = properties.get('aerialway', '')
-    service = properties.get('service')
+    service = properties.get('service', '')
+    ne_type = properties.get('type', '')
 
-    is_railway = railway in ('rail', 'tram', 'light_rail', 'narrow_guage', 'monorail')
+    is_railway = railway in (
+        'rail', 'tram', 'light_rail', 'narrow_guage', 'monorail')
 
-    if highway == 'motorway':
-        sort_val += 24
+    if (highway == 'motorway' or
+            ne_type in ('Major Highway', 'Beltway', 'Bypass')):
+        sort_val += 44
     elif is_railway:
-        sort_val += 23
-    elif highway == 'trunk':
-        sort_val += 22
-    elif highway == 'primary':
-        sort_val += 21
+        sort_val += 43
+    elif highway == 'trunk' or ne_type == 'Secondary Highway':
+        sort_val += 42
+    elif highway == 'primary' or ne_type == 'Road':
+        sort_val += 41
     elif highway == 'secondary' or aeroway == 'runway':
-        sort_val += 20
-    elif highway == 'tertiary' or aeroway == 'taxiway':
-        sort_val += 19
+        sort_val += 40
```

Comment:
77, though runways and highways will be styled enough differently – should they get their own sort_val?